class/defaults.yml

```diff
@@ -37,6 +37,26 @@ parameters:
           type: fluentd
           fluentd: {}
 
+    clusterLogForwarding:
+      enabled: false
+      spec:
+        pipelines:
```

Comment:
Interesting, makes a lot of sense for keeping the neccessary defaults. Will look into that.

Reply:
I looked into the `makeMergeable` thing, sadly it can't merge the arrays the way i wanted.
But as i played around with different configurations, i found out that i dont have to. If a user want to forward, lets say, the `infra` logs; he can just add an additional pipeline entry which selects the `infra` logs to the `.spec`, that way he can even add extra labels that are not needed for our internal elasticsearch.

Now i just merge the needed entries for normal operation into the spec.